kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/templates/testclients/TestClientsJobTemplates.java

```diff
@@ -51,25 +57,265 @@ public static JobBuilder defaultAdminClientJob(String jobName, List<String> args
+                        .withEnv(testClientsProducerEnvVars(bootstrap, topicName, numOfMessages, message))
+                        .withSecurityContext(jobsSecurityContext())
+                        .build())
+                .endSpec()
+                .endTemplate()
+                .endSpec();
+    }
+
+    /**
+     * Default test client consumer job builder.
+     *
+     * @param jobName the job name
+     * @param bootstrap the bootstrap
+     * @param topicName the topic name
+     * @param numOfMessages the num of messages
+     * @return the job builder
+     */
+    public static JobBuilder defaultTestClientConsumerJob(String jobName, String bootstrap, String topicName, int numOfMessages) {
+        return baseClientJob(jobName)
+                .editSpec()
+                .editTemplate()
+                .editSpec()
+                .withContainers(new ContainerBuilder()
+                        .withName("test-client-consumer")
+                        .withImage(Constants.TEST_CLIENTS_IMAGE)
+                        .withImagePullPolicy(Constants.PULL_IMAGE_IF_NOT_PRESENT)
+                        .withEnv(testClientsConsumerEnvVars(bootstrap, topicName, numOfMessages))
+                        .withSecurityContext(jobsSecurityContext())
+                        .build())
+                .endSpec()
+                .endTemplate()
+                .endSpec();
+    }
+
+    /**
+     * Default kcat job builder.
+     *
+     * @param jobName the job name
+     * @param args the args
+     * @return the job builder
+     */
+    public static JobBuilder defaultKcatJob(String jobName, List<String> args) {
+        return baseClientJob(jobName)
+                .editSpec()
+                .editTemplate()
+                .editSpec()
+                .withContainers(new ContainerBuilder()
+                        .withName("kcat")
+                        .withImage("edenhill/kcat:1.7.1")
+                        .withImagePullPolicy(Constants.PULL_IMAGE_IF_NOT_PRESENT)
+                        .withArgs(args)
+                        .withSecurityContext(jobsSecurityContext())
                         .build())
-                .withRestartPolicy("Never")
                 .endSpec()
                 .endTemplate()
                 .endSpec();
     }
+
+    /**
+     * Default kafka go producer job builder.
+     *
+     * @param jobName the job name
+     * @param bootstrap the bootstrap
+     * @param topicName the topic name
+     * @return the job builder
+     */
+    public static JobBuilder defaultKafkaGoProducerJob(String jobName, String bootstrap, String topicName) {
+        return baseClientJob(jobName)
+                .editSpec()
+                .editTemplate()
+                .editSpec()
+                .withContainers(new ContainerBuilder()
+                        .withName("kafka-go-producer")
+                        .withImage("ppatierno/kafka-go-producer:latest")
```

Comment:
Might https://github.com/birdayz/kaf be a better choice?  We are already using it elsewhere.  We build our own image: quay.io/kroxylicious/kaf

Reply:
Done. Thanks for the suggestion